### PR TITLE
[WIP] PR for SRVLOGIC-167: Update Serverless Logic Doc for TP1 (1.30.0)

### PR DESCRIPTION
**Pull request by kaldesai**
Tracking JIRA: https://issues.redhat.com/browse/SRVLOGIC-167

### CODE REVIEW

**Comment by caponetto** [comment]
Hi @kaldesai, thanks for the PR!
I've noticed that the `Serverless Logic Web Tools` section is missing. Its contents are being shown under `Chrome extension for GitHub` section.

**Review by krisv — APPROVED** [review]

**Comment by kaldesai** [comment]
> Hi @kaldesai, thanks for the PR! I've noticed that the `Serverless Logic Web Tools` section is missing. Its contents are being shown under `Chrome extension for GitHub` section.

Hello @caponetto, I have included the "Serverless Logic Web Tools" section now. Please take a look and let me know.

**Comment by kaldesai** [comment]
@MarianMacik and @ricardozanini can you please review this?

File: modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc [hunk 1]
```diff
@@ -39,14 +39,14 @@ The Chrome extension for Serverless Workflow editor is successfully installed.
...
 
 After installing the Chrome extension for the Serverless Workflow editor, you can use the editor to edit and view the workflow files in GitHub.
 
 .Prerequisites
 * You have installed the Serverless Workflow editor extension in Chrome. 
 +
-For more information, see <<proc-install-chrome-extension-sw-editor, Installing the Chrome GitHub extension for Serverless Workflow editor>>.
+For more information, see <<proc-install-chrome-extension-sw-editor, Installing the Chrome extension for Serverless Workflow editor on GitHub>>.
 
 .Procedure
```
Comment (caponetto) [inline modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:51]: Is it possible to update the image `image::tooling/serverless-workflow-editor/swf-editor-in-github.png` to [this one](https://github.com/apache/incubator-kie-kogito-docs/blob/main/serverlessworkflow/modules/ROOT/assets/images/tooling/serverless-workflow-editor/swf-editor-in-github-readonly.png)?

**Review by caponetto — COMMENTED** [review]
Thanks @kaldesai! Please consider these 2 final comments.

File: modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc [hunk 2]
```diff
@@ -27,19 +27,19 @@ image::tooling/serverless-workflow-editor/swf-editor-vscode-extension-page.png[]
 .Procedure
 . You can install the VS Code extension for Serverless Workflow editor using one of the following methods:
 
-* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[Serverless Workflow Editor] extension page in Visual Studio Marketplace and click *Install*.
+* *Go to Visual Studio Marketplace*: Go to link:{kogito_swf_editor_vscode_marketplace_url}[{serverless_workflow_vscode_extension_name}] extension page in Visual Studio Marketplace and click *Install*.
```
[the remark this answers is not in the document]
Comment (kaldesai) [reply modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc:30]: Done!

File: modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc (hunk 1 above)
Comment (kaldesai) [reply modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:51]: Done!

File: modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (caponetto) [reply modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc:30]: I'm still seeing the placeholder on this page. Am I missing something?

image

[the remark this answers is not in the document]
Comment (kaldesai) [reply modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc]: Can you please check now? I changed it. Forgot to add.

[the remark this answers is not in the document]
Comment (caponetto) [reply modules/serverless-logic/pages/tooling/serverless-workflow-editor/swf-editor-vscode-extension.adoc]: We're almost there. There is a curly brace left over.

image

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -35,7 +35,7 @@
 //***** xref:serverless-logic:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-workflow-definition-page.adoc[Workflow Definitions in Kogito Serverless Workflow Tools extension]
 //***** xref:serverless-logic:tooling/quarkus-dev-ui-extension/quarkus-dev-ui-custom-dashboard-page.adoc[Dashboards in Kogito Serverless Workflow Tools extension]
 //**** xref:serverless-logic:tooling/kn-plugin-workflow-overview.adoc[{product_name} plug-in for Knative CLI]
-**** xref:serverless-logic:tooling/serverless-logic-web-tools/serverless-logic-web-tools-overview.adoc[{serverless_logic_web_tools_name}]
+**** xref:serverless-logic:tooling/serverless-logic-web-tools/serverless-logic-web-tools-overview.adoc[KIE Serverless Workflow Editor]
```
Comment (caponetto) [inline modules/ROOT/nav.adoc:38]: I think here we need to keep `{serverless_logic_web_tools_name}`

**Comment by domhanak** [comment]
@kaldesai @ricardozanini imho this seems to be missing some guides, I guess backports are not merged on our repo

**Comment by kaldesai** [comment]
> @kaldesai @ricardozanini imho this seems to be missing some guides, I guess backports are not merged on our repo

@ricardozanini, can you guide me with this?

**Review by caponetto — APPROVED** [review]
Thanks, @kaldesai! The tooling part looks good to me.

**Comment by kaldesai** [comment]
> Thanks, @kaldesai! The tooling part looks good to me.

Thank you so much, @caponetto for reviewing and verifying it. Much appreciated!

**Comment by ricardozanini** [comment]
> > @kaldesai @ricardozanini imho this seems to be missing some guides, I guess backports are not merged on our repo
> 
> @ricardozanini, can you guide me with this?

I'm sorry, I just merged the last PRs to 1.40.x branch @kaldesai. It seems that I missed the notifications after the migration. Can you please include them in this PR (you can rebase your kogito-docs local and add the changed files to this branch by copying and pasting here)?

- https://github.com/apache/incubator-kie-kogito-docs/pull/479
- https://github.com/apache/incubator-kie-kogito-docs/pull/475

@domhanak am I missing something else?

File: antora.yml [hunk 4]
```diff
@@ -12,22 +12,22 @@ asciidoc:
...
-    kogito_version_redhat: 1.38.0.Final-redhat-00002
+    kogito_version_redhat: 1.40.0.Final-redhat-00009
     operator_name: Kogito Serverless Workflow Operator
     quarkus_platform: com.redhat.quarkus.platform
     kogito_sw_ga: >-
       org.kie.kogito:kogito-quarkus-serverless-workflow
-    quarkus_version: 2.13.7.Final-redhat-00003
-    quarkus_platform_version: 2.13.7.Final-redhat-00003
+    quarkus_version: 2.13.8.Final-redhat-00005
+    quarkus_platform_version: 2.13.8.SP2-redhat-00001
```
Comment (MarianMacik) [inline antora.yml:21]: These are SP1 versions.

File: antora.yml [hunk 5]
```diff
@@ -12,22 +12,22 @@ asciidoc:
...
     java_min_version: 11+
     maven_min_version: 3.8.6
     graalvm_min_version: 22.3.0
     spec_version: 0.8
     vscode_version: 1.66.0
-    kn_cli_version: 0.28.0
+    kn_cli_version: 0.30.3
```
Comment (MarianMacik) [inline antora.yml:27]: @domhanak This is version from the TP1. I guess the SP1 has the same one.

Comment (domhanak) [reply antora.yml:27]: Yes, the  correct version  of quarkus used in kn cli is injected during the openshift-knative build process

**Pull request closed without merging**